sample-keys/README.md

```diff
@@ -0,0 +1,127 @@
+# TLS and mTLS for Tornjak
+
+In order to enable TLS and mTLS connection for Tornjak, a server must be configured with certificate and key information. This document outlines the following steps to try out the feature:
+
+1. [Create the relevant TLS/mtls files](#create-the-relevant-tls-files)
+2. [Configure the Tornjak server](#configure-the-tornjak-server)
+3. [Making calls to Tornjak Backend](#making-calls-to-tornjak-backend)
+
+## Create the relevant TLS/mTLS files
+
+The relevant certificates and keys first require a CA to sign a certificate. 
+
+Note that sample certificates and keys are included in publicly provided image `ghcr.io/spiffe/tornjak-backend` within the `sample-keys` directory. The below steps are available to OPTIONALLY create your own local certificates and keys. 
+
+<details><summary><b> 🔴 [Click] Create your own CA and certificate/key pair</b></summary>
+
+### Create a CA
+
```

Comment:
```suggestion

You can bring your own CA to use for signing the certificates. Replace the CA and key in `CA/` directory with yours.
```

Reply:
We should probably also show how to create the keys. Here is the complete document: https://github.com/IBM/trusted-service-identity/blob/main/docs/spire-security.md